zulip_bots/zulip_bots/bots/salesforce/doc.md

```diff
@@ -0,0 +1,52 @@
+# Salesforce bot
+
+The Salesforce bot can get records from your Salesforce database.
+It can also show details about any Saleforce links that you post.
+
+## Setup
+
+1. Create a user in Saleforce that the bot can use to access Salesforce.
+Make sure it has the appropriate permissions to access records.
+2. In `salesforce.conf` paste the Saleforce `username`, `password` and
+`security_token`.
+3. Run the bot as explained [here](https://zulipchat.com/api/running-bots#running-a-bot)
+
+## Configuration (Advanced)
+
+The bot has been designed to be able to configure custom commands and objects.
```

Comment:
Can you add a little more detail to the purpose of the custom configuration? E.g. for what type of user it would be useful, what Commands and Object Types are, etc.